pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -56,18 +64,33 @@ func TestClusterWideMode(t *testing.T) {
 		t.NewSubTest("default namespace selector").Run(func(t test.TestHelper) {
 			t.Log("Check whether namespaces with the label istio-injection=enabled become members automatically")
 
-			t.LogStep("Create 50 member namespaces")
-			createMemberNamespaces(t, 50)
+			t.LogStep("Create 5 member namespaces")
+			createMemberNamespaces(t, 5)
 
 			t.LogStep("Wait for SMMR to be Ready")
 			oc.WaitSMMRReady(t, meshNamespace)
 
-			t.LogStep("Check whether the SMMR shows all 50 namespaces as members")
-			shell.Execute(t,
-				fmt.Sprintf("oc -n %s get smmr default", meshNamespace),
-				require.OutputContains("50/50",
-					"all 50 namespaces are members",
-					"expected SMMR to show 50 member namespaces, but that wasn't the case"))
+			t.LogStep("Check whether the SMMR shows the 5 namespaces created as members")
+			membersList := []string{"member-0", "member-1", "member-2", "member-3", "member-4"}
+			assertMembers(t, meshNamespace, membersList)
+
+		})
+
+		t.NewSubTest("RoleBindings verification").Run(func(t test.TestHelper) {
+			t.Log("Related to OSSM-3468")
+			t.LogStep("Check that Rolebindings are not created in the member namespaces")
+			retry.UntilSuccess(t, func(t test.TestHelper) {
+				oc.Get(t, "member-0", "rolebindings", "",
+					assert.OutputContains("prometheus-istio-system",
```

Comment:
This role binding was removed, but then I [reverted](https://github.com/maistra/istio-operator/pull/1194) that change, so now this test should fail. Actually, since we will no longer use our prometheus, we can revert my commit to remove that role binding back, but it's too late and it has to be postponed to 2.4.1.
Anyway, this assertion should fail and you will have to remove it for now.